tools/serialize_nano_core/src/main.rs

```diff
@@ -0,0 +1,30 @@
+//! Tool that creates a serialized representation of the symbols in the `nano_core` object file.
+
+mod parse;
+
+use mod_mgmt::serde::SerializedCrate;
+use std::io::Write;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let arg = &std::env::args().collect::<Vec<String>>()[1];
```

Comment:
nit:  if you just want the first arg, you can do `args().nth(1)` instead of collecting them all into a vector